### ISSUE TITLE
Secondary badges do not meet color contrast requirements

### ISSUE THREAD

**Issue by McManning** [osuresearch/ui#88]
•	Element has insufficient color contrast of 3.03 (foreground color: #ffffff, background color: #1aaa55, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1
•	[WCAG 1.4.](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum)3 Contrast (Minimum)
•	Recommendation: Darken the text to meet the requirement.

**Comment by helloLilyX** [comment]
The background color should as #757575 in
`.badge-secondary {
    color: #fff;
    background-color: #757575;
}`

**Comment by McManning** [comment]
I misreported this one - it's not the badge component but the `Filters.Active` badges:

![image](https://github.com/osuresearch/ui/assets/1057607/d4c80ea6-03cf-4e39-87ab-6481a310963d)

If you go to the search components example and select some states as filters, some badges will appear. Those are the active filter components:

https://osuresearch.github.io/ui/v4/#/Search%20Components?id=section-filters

**Comment by McManning** [comment]
This is also specifically a problem with the hover state for them: 
![image](https://github.com/osuresearch/ui/assets/1057607/d3e2aecd-68a4-485e-8868-5cf5ff5d1835)
